service/collector/app.go

```diff
@@ -146,19 +153,59 @@ func (c *App) collectAppStatus(ctx context.Context, ch chan<- prometheus.Metric)
 	return nil
 }
 
-func (c *App) getTeam(ctx context.Context, app v1alpha1.App) (string, error) {
-	name := key.AppCatalogEntryName(app.Spec.Catalog, app.Name, app.Spec.Version)
+func (a *App) getOwningTeam(ctx context.Context, app v1alpha1.App, ownersYAML string) (string, error) {
+	owners := []owner{}
+
+	err := yaml.Unmarshal([]byte(ownersYAML), &owners)
+	if err != nil {
+		// If the YAML in the owners annotation is invalid log the error and
+		// fall back to trying the team annotation.
+		a.logger.Errorf(ctx, err, "could not parse owners YAML for app %q", key.AppName(app))
```

Comment:
Since this is an exporter already, what do you think to expose these cases as metrics?

Reply:
Yes good idea. As it is we're unlikely to see these logs.

Reply:
When I looked at this again I'm not sure adding another metric makes sense. Worst case if this goes wrong the team will default to batman and we can investigate. 

If this happens often we can add the metric. @tomahawk28 WDYT?

Reply:
Sure, no problem 👍